### PR TITLE
Switch compression off

### PR DESCRIPTION
**Pull request by kberezin-nshl**

### CODE REVIEW

**Review by chrisfesler — APPROVED** [review]

File: packages/renderer/src/index.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ const pdf = (initialValue, cache) => {
 
   if (initialValue) updateContainer(initialValue);
 
-  const render = async (compress = true) => {
+  const render = async (compress = false) => {
```
Comment (pwalker) [inline packages/renderer/src/index.js:34]: Is it possible to pass this through as an option from `renderToBuffer` or similar methods?

Comment (chrisfesler) [reply packages/renderer/src/index.js:34]: why would we want to do that if it slows render by 50%?

Comment (chrisfesler) [reply packages/renderer/src/index.js:34]: not render -- slows the _entire request_ down by 50%

**Review by awhipple — APPROVED** [review]

**Comment by realark** [comment]
Peter pulled this work into here: https://github.com/nutshelllabs/react-pdf/pull/14

**Pull request closed without merging**